### PR TITLE
Move to Xamarin.Forms

### PR DESCRIPTION
**Pull request by hickoryeyes**
**Changes from original code:**
- Now using Xamarin.Forms.GoogleMaps for maps _(Fun fact: that's what CityBee uses)_
- Edited the Xamarin.Forms template to replace all <Item> objects with our locations from Xamarin.Android, the template already had the option to add items
- Left most of the registration and login

**Why moving is worth the trouble:**
- Easily add a flyout or tabbed pages
- Icons on text entries, show/hide passwords, etc
- More plugins and useful stuff found on GitHub
- More tutorials and Stackoverflow answers
- Clearer app structure (views, models)

If you were to need any help navigating the change, I'm here to help!

### CODE REVIEW

**Review by MigleMic — APPROVED** [review]
Looks and works great!
Everything was migrated from xamarin to xamarin.forms 
Everything was easy to follow, now cs file names are more correct, like from regActivity to RegisterPage.
I think we should consider moving to xamarin.forms

**Comment by Waltzerizm** [comment]
Looks and works good to me, great job:-)

**Review by Waltzerizm — APPROVED** [review]
Looks and works good to me, great job:-)

File: Encountify/Views/ItemsPage.xaml.cs [hunk 1]
```diff
@@ -0,0 +1,23 @@
+﻿using Encountify.ViewModels;
```
Comment (Mdominykas) [inline Encountify/Views/ItemsPage.xaml.cs:1]: If I understand correctly then ItemsPage is Page with LocationList. If that's the case I believe it should be rename to more clear name.

**Review by Mdominykas — CHANGES_REQUESTED** [review]
Looks good, though I really believe that file name mentioned in comment should be changed

File: Encountify/Views/ItemsPage.xaml.cs (hunk 1 above)
Comment (hickoryeyes) [reply Encountify/Views/ItemsPage.xaml.cs]: Sure thing!

**Pull request merged**